### PR TITLE
Added example showing target=release_debug

### PR DESCRIPTION
**Pull request by raould**
see https://github.com/godotengine/godot/issues/2095

### CODE REVIEW

File: reference/introduction_to_the_buildsystem.rst [hunk 1]
```diff
@@ -86,6 +86,20 @@ Example for using 4 cores:
 
     user@host:~/godot$ scons platform=x11 -j 4
 
+The default build is not heavily optimized, to allow
```
Comment (akien-mga) [inline reference/introduction_to_the_buildsystem.rst]: Actually not optimized at all, it's a debug build :)

Could you make a new section for that? You put it at the right place, but it shouldn't be under "Platform selection", best make a new "Optimized build" section.

**Comment by raould** [comment]
ok i will update as soon as i can :-)

**Comment by akien-mga** [comment]
It seems you rebased but did not do the changes mentioned?

**Comment by akien-mga** [comment]
> It seems you rebased but did not do the changes mentioned?

Ping.

**Comment by raould** [comment]
working on it now...

**Pull request closed without merging**